robocop/checkers/duplications.py

```diff
@@ -96,6 +96,8 @@ def visit_VariableSection(self, node):  # noqa
         self.generic_visit(node)
 
     def visit_Variable(self, node):  # noqa
+        if node.name == '':
```

Comment:
``if not node.name`` could be better, if name is None (can't think of example when it's None but just in case)